polaris.shopify.com/lib/ga/index.js

```diff
@@ -0,0 +1,15 @@
+export const GA_TRACKING_ID = process.env.NEXT_PUBLIC_GA_ID
```

Comment:
Could you possibly move this folder under src? This way all of the src code is located within the same root dir?

Reply:
Done!